content/api/commands/clearalllocalstorage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: clearAllLocalStorage
+---
+
+Clear localStorage data for all origins with which the test has interacted.
+
+<Alert type="warning">
+
+Cypress automatically runs this command _before_ each test to prevent state from
+being shared across tests. You shouldn't need to use this command unless you're
+using it to clear localStorage inside a single test.
+
+</Alert>
+
+## Syntax
+
+```javascript
+cy.clearAllLocalStorage()
+cy.clearAllLocalStorage(options)
+```
+
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```javascript
+cy.clearAllLocalStorage()
+```
+
+### Arguments
+
+**<Icon name="angle-right"></Icon> options** **_(Object)_**
+
+Pass in an options object to change the default behavior of
+`cy.clearAllLocalStorage()`.
+
+| Option | Default | Description                                                                              |
+| ------ | ------- | ---------------------------------------------------------------------------------------- |
+| `log`  | `true`  | Displays the command in the [Command log](/guides/core-concepts/cypress-app#Command-Log) |
+
+### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
+
+- `cy.clearAllLocalStorage()` yields `null`.
+- `cy.clearAllLocalStorage()` cannot be chained further.
+
+## Examples
+
+### Clear all localStorage
+
+```javascript
+cy.clearAllLocalStorage()
+```
```

Comment:
I'm not sure this section adds value since it's the same as the usage example.

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/4876/commits/1b31d3d09ba10bea4afcd9cdc34af0fd7480bd79